src/components/staking/LiValidator.vue

```diff
@@ -35,9 +35,12 @@
         :alt="`validator logo for ` + validator.name"
       />
       <div class="validator-info">
-        <h3 class="li-validator-name">
+        <h3 v-if="!validator.name.startsWith('0x')" class="li-validator-name">
```

Comment:
i don't think we should have special network-specific logic here. 

we could pass 0x addresses into bech32 where the truncation could happen. we can open a separate ticket to refactor the name of the component so it's more generic.